src/include/rfac_def.h

```
@@ -108,6 +114,79 @@ struct rfspot
+  RFAC_INVALID_ARGUMENT,
+  RFAC_IV_NOT_EQUIDISTANT,
+  RFAC_ALLOCATION_ERROR,
+  RFAC_ESTEP_TOO_SMALL,
+  RFAC_VI_TOO_SMALL,
+  RFAC_INVALID_NUMBER_OF_ENERGIES,
+  RFAC_INVALID_NUMBER_OF_BEAMS,
+  RFAC_BAD_IV_DATA
+} rfac_error;
+
+#ifndef END_OF_GROUP_ID
+#define END_OF_GROUP_ID -1
```

Clang (14.0.6)

[warning] macro replacement list should be enclosed in parentheses

(bugprone-macro-parentheses)